### PR TITLE
chore(website): Add new guides section

### LINKED ISSUE #2115: Source/sink matrix of guides

### PR DESCRIPTION
**Pull request by binarylogic**
This pull request adds a new `/guides` section. This is a markdown driven section built for managed guides and tutorials so that we can get them out of our docs. A few important things to note:

1. Guides have categories represented by the folder they are in.
2. A source/sink matrix of integration guides is automatically created.
3. Installation instructions were broken down in to more logical components so that we could generate them accurately across platforms.

Closes #2115

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -1,65 +1,65 @@
 - name: "domain: blog"
-  description: Anything blog related
+  description: Anything  related
```
Comment (Hoverbear) [inline .github/labels.yml]: Please check these @binarylogic

Comment (binarylogic) [reply .github/labels.yml]: Thanks, fixed.

**Comment by Hoverbear** [comment]
On the guide pages it'd be nice to make the source a selector, like the sink:

image

**Comment by Hoverbear** [comment]
On a Windows user agent this guide is still showing instructions for Linux. :(

image

**Comment by Hoverbear** [comment]
Generally this looks awesome, @binarylogic . :)

**Comment by binarylogic** [comment]
> On the guide pages it'd be nice to make the source a selector, like the sink:

Done!

> Could these be done via vector generate?

I don't think this makes sense for the guides because users need to modify required values before creating the file. It's also not obvious what the options are when reading the guides.

> On a Windows user agent this guide is still showing instructions for Linux. :(

I'll create an issue for this.

**Pull request merged**